neon_llm_vllm/vllm.py

```diff
@@ -89,24 +121,29 @@ def get_sorted_answer_indexes(self, question: str, answers: List[str], persona:
         sorted_items_indexes = [x[0] for x in sorted_items]
         return sorted_items_indexes
 
-    def _call_model(self, prompt: List[Dict[str, str]]) -> str:
+    def _call_model(self, prompt: str) -> str:
         """
-            Wrapper for ChatGPT Model generation logic
+            Wrapper for VLLM Model generation logic
             :param prompt: Input messages sequence
             :returns: Output text sequence generated by model
         """
 
-        response = openai.ChatCompletion.create(
+        response = self.model.completions.create(
             model=self.llm_model_name,
-            messages=prompt,
+            prompt=prompt,
             temperature=0,
             max_tokens=self.max_tokens,
+            extra_body={
```

Comment:
maybe set these properties in `__init__` with ability to override them from configs